### PR TITLE
feat: Spanish locale 🇪🇸🇪🇸🇪🇸

### PR DESCRIPTION
**Pull request by abrahammurciano**
## Description

Fixes # 

### Type of change

- [ ] Bug fix (added a non-breaking change which fixes an issue)
- [ ] New feature (added a non-breaking change which adds functionality)
- [ ] Updated documentation (updated the readme, templates, or other repo files)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

- [ ] Tested locally with a valid username
- [ ] Tested locally with an invalid username
- [ ] Ran tests with `composer test`
- [ ] Added or updated test cases to test new features

## Checklist:

- [ ] I have checked to make sure no other [pull requests](https://github.com/DenverCoder1/github-readme-streak-stats/pulls?q=is%3Apr+sort%3Aupdated-desc+) are open for this issue
- [ ] The code is properly formatted and is consistent with the existing code style
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for contributing :tada:

File: src/translations.php [hunk 1]
```diff
@@ -3,14 +3,17 @@
 return [
     "currentStreak" => [
         "en" => "Current Streak",
+        "es" => "Racha Actual",
         "ja" => "現在のストリーク",
     ],
     "totalContributions" => [
         "en" => "Total Contributions",
+        "es" => "Número de Contribuciones",
```
Comment (DenverCoder1) [inline src/translations.php:11]: I think we can change this to make it shorter as discussed on Discord

```suggestion
        "es" => "Todas Contribuciones",
```

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]

**Review by DenverCoder1 — APPROVED** [review]

**Comment by abrahammurciano** [comment]
It's "Presente"

**Comment by DenverCoder1** [comment]
Do you want to make a PR or should I just add it?

**Comment by abrahammurciano** [comment]
Just add it, I'm busy rn and it's such a small thing

**Comment by DenverCoder1** [comment]
Sounds good, thanks

**Pull request merged**